templates/download/mediatek-genio/tab-3.html

```diff
@@ -15,6 +15,28 @@ <h2 class="u-hide--small">Genio 350 EVK</h2>
       }}
     </div>
   </div>
+  <div class="row p-strip u-no-padding--top">
+    <hr class="col-9">
+    <div class="col-3">
+      <h3 class="p-heading--5 u-no-padding--top">Ubuntu Desktop 22.04 LTS</h3>
```

Comment:
Instead of this `u-no-padding--top` the margin should be removed from hr element above.